src/models.ts

```diff
@@ -0,0 +1,19 @@
+import { K8sModel } from "@openshift-console/dynamic-plugin-sdk";
+import { CRONTAB_APIGROUP, CRONTAB_APIVERSION } from "./const";
+
+export const CronTabModel: K8sModel = {
```

Comment:
So I think there should be a way how we could avoid defining the model statically in the code. We had a discussion about it with @spadgett when the crontab-plugin was implemented. 
Unfortunately the `console.flag/model` is defining `ExtensionK8sModel` type for a model. 
@spadgett thoughts ?

Reply:
So this was a misunderstanding on my part as I was duplicating code from console.  I removed `[cronTabGroupVersionKind](src/utils/utils.ts)` and replaced it with the model in order to use `getGroupVersionKindForModel` from the SDK.  I will undo that change.